src/components/CardEditor/CardEditForm/CardEditForm.test.jsx

```diff
@@ -0,0 +1,73 @@
+import {
+  CARD_DIMENSIONS,
+  CARD_SIZES,
+} from '../../../constants/LayoutConstants';
+
+import { getCardSizeText, handleSubmit } from './CardEditForm';
+
+const mockSetError = jest.fn();
+const mockOnChange = jest.fn();
+const mockSetShowEditor = jest.fn();
+afterEach(() => {
+  jest.clearAllMocks();
+});
+describe('CardEditForm', () => {
+  describe('getCardSizeText', () => {
+    const i18n = {
+      cardSize_SMALL: 'Small',
+      cardSize_SMALLWIDE: 'Small wide',
+      cardSize_MEDIUM: 'Medium',
+      cardSize_MEDIUMTHIN: 'Medium thin',
+      cardSize_MEDIUMWIDE: 'Medium wide',
+      cardSize_LARGE: 'Large',
+      cardSize_LARGETHIN: 'Large thin',
+      cardSize_LARGEWIDE: 'Large wide',
+    };
+    it('should return correct size and dimensions', () => {
+      expect(getCardSizeText(CARD_SIZES.SMALL, i18n)).toEqual(
+        `${i18n.cardSize_SMALL} (${CARD_DIMENSIONS.SMALL.lg.w}x${CARD_DIMENSIONS.SMALL.lg.h})`
+      );
+      expect(getCardSizeText(CARD_SIZES.SMALLWIDE, i18n)).toEqual(
+        `${i18n.cardSize_SMALLWIDE} (${CARD_DIMENSIONS.SMALLWIDE.lg.w}x${CARD_DIMENSIONS.SMALLWIDE.lg.h})`
+      );
+      expect(getCardSizeText(CARD_SIZES.MEDIUM, i18n)).toEqual(
+        `${i18n.cardSize_MEDIUM} (${CARD_DIMENSIONS.MEDIUM.lg.w}x${CARD_DIMENSIONS.MEDIUM.lg.h})`
+      );
+      expect(getCardSizeText(CARD_SIZES.MEDIUMTHIN, i18n)).toEqual(
+        `${i18n.cardSize_MEDIUMTHIN} (${CARD_DIMENSIONS.MEDIUMTHIN.lg.w}x${CARD_DIMENSIONS.MEDIUMTHIN.lg.h})`
+      );
+      expect(getCardSizeText(CARD_SIZES.MEDIUMWIDE, i18n)).toEqual(
+        `${i18n.cardSize_MEDIUMWIDE} (${CARD_DIMENSIONS.MEDIUMWIDE.lg.w}x${CARD_DIMENSIONS.MEDIUMWIDE.lg.h})`
+      );
+      expect(getCardSizeText(CARD_SIZES.LARGE, i18n)).toEqual(
+        `${i18n.cardSize_LARGE} (${CARD_DIMENSIONS.LARGE.lg.w}x${CARD_DIMENSIONS.LARGE.lg.h})`
+      );
+      expect(getCardSizeText(CARD_SIZES.LARGETHIN, i18n)).toEqual(
+        `${i18n.cardSize_LARGETHIN} (${CARD_DIMENSIONS.LARGETHIN.lg.w}x${CARD_DIMENSIONS.LARGETHIN.lg.h})`
+      );
+      expect(getCardSizeText(CARD_SIZES.LARGEWIDE, i18n)).toEqual(
+        `${i18n.cardSize_LARGEWIDE} (${CARD_DIMENSIONS.LARGEWIDE.lg.w}x${CARD_DIMENSIONS.LARGEWIDE.lg.h})`
+      );
+    });
+  });
+
+  // would like to do react-testing-library tests with this, but we're unable to render the actual editor,
```

Comment:
> we're unable to render the actual editor

Bummer, why is this?

Reply:
This may be due to how monaco-editor loads itself with a script tag. This is another use case for our cypress test. I had fiddled around with this for a bit trying to get jsdom to play nice but was taking too long.

Reply:
I didn't look too deep into it, but I noticed the editor not rendering in jest when I tried to target it. I then checked the snapshots and noticed it wasn't there either. I can take a deeper look as I fix the other snapshots today though

Reply:
That makes sense - yet another thing for #1203 haha